docs/zkapps/tutorials/05-common-types-and-functions.mdx

```diff
@@ -41,7 +41,7 @@ const num2 = UInt64.from(40);
 
 const num1EqualsNum2: Bool = num1.toUInt64().equals(num2);
 
-console.log(`num1 === num2: ${num1EqualsNum2.toString()}`);
+console.log(`num1 == num2: ${num1EqualsNum2.toString()}`);
```

Comment:
Quick question why you decided to show == over === in these console.logs ?

Reply:
ha, I didn't do that consciously, I copied over the code example code into the tutorial. So I guess this might be something that was fixed in the tutorial but not Evan's code. Will change it to `===` again!